docs/_includes/generated/rest_v1_dispatcher.html

```diff
@@ -2526,7 +2529,7 @@
       "type" : "array",
       "items" : {
         "type" : "object",
-        "id" : "urn:jsonschema:org:apache:flink:runtime:rest:messages:SubtaskExecutionAttemptDetailsInfo",
+        "id" : "urn:jsonschema:org:apache:flink:runtime:rest:messages:job:SubtaskExecutionAttemptDetailsInfo",
```

Comment:
it appears the api docs were a bit outdated; can you move this change and below ones into a separate commit?

Reply:
ok, will update it